apps/daimo-mobile/src/view/TabNav.tsx

```diff
@@ -154,7 +107,6 @@ export function TabNav() {
             ...TransitionPresets.ModalPresentationIOS,
             detachPreviousScreen: false,
             gestureResponseDistance: WINDOW_HEIGHT,
-            cardStyleInterpolator: errorModalInterpolatorIOS,
```

Comment:
🎉 

"when you have a problem, remove the problem"

looks great to me! nice simplification